assets/main-menu-dropdown.js

```diff
@@ -1,9 +1,11 @@
 $("[data-toggle='ecosystem-dropdown']").on("click", function() {
   toggleDropdown($(this).attr("data-toggle"));
+  $(this).children("a").toggleClass("with-right-white-arrow with-down-white-arrow")
 });
 
 $("[data-toggle='resources-dropdown']").on("click", function() {
   toggleDropdown($(this).attr("data-toggle"));
+  $(this).children("a").toggleClass("with-right-white-arrow with-down-white-arrow")
 });
```

Comment:
Since these two click functions are the same maybe you can look into refactoring. Also, you're missing a semicolon at the end of both toggleClass functions.